components/brave_today/browser/test/BUILD.gn

```diff
@@ -12,6 +12,7 @@ source_set("brave_news_unit_tests") {
     "//brave/components/brave_today/browser/brave_news_p3a_unittest.cc",
     "//brave/components/brave_today/browser/direct_feed_controller_unittest.cc",
     "//brave/components/brave_today/browser/feed_building_unittest.cc",
+    "//brave/components/brave_today/browser/html_parsing_unittest.cc",
```

Comment:
why is this file (BUILD.gn) in a different directory than the actual tests?

Reply:
@bridiver I think I originally saw the pattern from e.g. components/brave_wallet/browser/test/BUILD.gn